src/TestPlatform.TranslationLayer.nuspec

```diff
@@ -50,12 +48,124 @@
     <file src="net46\$Runtime$\Microsoft.TestPlatform.CommunicationUtilities.dll" target="lib\net46\" />
     <file src="net46\$Runtime$\Microsoft.TestPlatform.CoreUtilities.dll" target="lib\net46\" />    
```

Comment:
We can remove coreutilities as well? OM package will pull it in? Or may be make coreutilities a separate nuget package?